RecoTracker/LSTCore/src/alpaka/MiniDoublet.h

```diff
@@ -85,7 +85,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::lst {
     mds.outerLowEdgeY()[idx] = hits.lowEdgeYs()[outerHitIndex];
   }
 
-  ALPAKA_FN_ACC ALPAKA_FN_INLINE float isTighterTiltedModules(ModulesConst modules, uint16_t moduleIndex) {
+  ALPAKA_FN_ACC ALPAKA_FN_INLINE bool isTighterTiltedModules(ModulesConst modules, uint16_t moduleIndex) {
```

Comment:
this is incomplete; cf https://github.com/SegmentLinking/cmssw/pull/115/commits/fc792fd89dd4716c05f44c0fe9e2fdf21891e8db
I'm not sure if identical change is a conflict, hopefully not